jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/skaffold/SyncMapTask.java

```diff
@@ -77,7 +77,10 @@ public void listFilesAndTargets() {
 
       try {
         String syncMapJson =
-            PluginConfigurationProcessor.getSkaffoldSyncMap(configuration, projectProperties);
+            PluginConfigurationProcessor.getSkaffoldSyncMap(
+                configuration,
+                projectProperties,
+                jibExtension.getSkaffold().getSync().getExcludes());
```

Comment:
So where do includes come in?

Reply:
Nm, I forgot the distinction between sync and watch

Reply:
like when are they useful?

for user configured modifications

`some resource` -> `[user-task]` -> `output` -> `[jib-build]` -> `container`

normally jib would just watch `output`, but in reality it should watch `some resource`,
so user includes `some resource`, excludes `output`

Reply:
There is maybe someway to intelligently do this? like decide if `output` was an actual output to some other task, but I think it's not straightforward.